Chummer/Backend/Equipment/Gear.cs

```diff
@@ -182,6 +183,10 @@ private void ChildrenOnCollectionChanged(object sender, NotifyCollectionChangedE
             if (!objXmlGear.TryGetMultiLineStringFieldQuickly("altnotes", ref _strNotes))
                 objXmlGear.TryGetMultiLineStringFieldQuickly("notes", ref _strNotes);
 
+            String sNotesColor = ColorTranslator.ToHtml(ColorManager.HasNotesColor);
+            objXmlGear.TryGetStringFieldQuickly("notesColor", ref sNotesColor);
+            _colNotes = ColorTranslator.FromHtml(sNotesColor);
+
```

Comment:
Is this necessary in a method that gets called on/near creation of the gear? Would there need to be data-definable colors for specific gear?